sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -72,6 +91,15 @@ def test_check_estimator():
     # check that fit does input validation
     msg = "TypeError not raised by fit"
     assert_raises_regex(AssertionError, msg, check_estimator, BaseBadClassifier)
+    # check that sample_weights in fit accepts pandas.Series type
+    try:
+        from pandas import Series
```

Comment:
Here, I check if I am able to import pandas before the new estimator check (skip if not). Is there a better way to handle this? 
pep8 complained about an unused import in my previous commit because I would just do a try-catch for an ImportError but would do nothing with the import itself. Now, I am using it in `msg` to avoid that lint error. Should I be handling this in a different way?